BUILD.md

```diff
@@ -242,7 +242,7 @@ It patches their CMake to correctly import its dependencies.
     Single-config generators:
 
     ```
-    cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release ..
+    cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release -Dxrpld=TRUE ..
```

Comment:
```suggestion
    cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release -Dxrpld=TRUE -Dtests=TRUE ..
```

There's also a `cmake` command line under [Coverage report](https://github.com/XRPLF/rippled/blob/fd9c56c02516ef86e6ded145d6ee8c8497547698/BUILD.md#coverage-report) where it would be good to add these options.

Reply:
I know it doesn't actually matter as long as it's defined and something not "falsey" but the (very few) other bools in this file are defined as `ON`.

All the `rippled` source specifies CMake bools as `ON`.

I know Conan uses `True` but that's because that's the Python boolean and the GitHub Action sets `TRUE` probably because John was just in Conan mode.

Reply:
Good point. I will use `ON`